_includes/markdown/Markup.md

```diff
@@ -285,10 +285,128 @@ Combining automated and manual testing practices allows 10up to maintain a high
 <h2 id="progressive-enhancement" class="anchor-heading">Progressive Enhancement {% include Util/top %}</h2>
 Progressive enhancement means building a website that is robust, fault tolerant, and accessible. Progressive enhancement begins with a baseline experience and builds out from there, adding features for browsers that support them. It does not require us to select supported browsers or revert to table-based layouts. Baselines for browser and device support are set on a project-by-project basis.
 
-At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible. For example, browser support for SVG has not yet reached 100%. When using SVG you should always provide a fallback such as a PNG image for browsers that do not support vector graphics.
+At 10up, we employ progressive enhancement to ensure that the sites we build for our clients are accessible to as many users as possible.
 
 ### Polyfills
 When writing markup that does not have wide browser support, using polyfills can help bring that functionality to those older browsers. Providing support for older browsers is incredibly important to the business objectives of our clients. In an effort to prevent code bloat, we only provide polyfills for features that are functionally critical to a site.
 
 ### Feature Detection
 At 10up, the concept of feature detection is used to test browser support for new features that do not yet have full support across the board. The concept of feature detection is to test if a feature is supported by the browser and if not supported, conditionality run code to provide a similar experience with browsers that do support the feature. While popular [feature detection libraries](https://modernizr.com/) exist, there are [feature detection techniques](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection#JavaScript) for JavaScript and [@supports](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) at-rule for CSS that can be utilized.
+
+<h2 id="svg" class="anchor-heading">SVG {% include Util/top %}</h2>
+<abbr title="Scaleable Vector Graphics">SVG</abbr> has become a prevalent means for displaying rich vector graphics. Here are a few known benefits of SVG:
+
+* __Scalability__ - They look great on retina displays and at any size, i.e. they're resolution independent.
+* __File Size__ - Small file size and compresses well.
+* __Styling__ - Manipulate fill, stroke, and even animate.
+
+
+### Best Practices &amp; Common Usage / Pitfalls
+
+#### I've got an SVG on my page, now why is it so massive?
```

Comment:
All of these sections are more like mini-tutorials rather than best practice. Instead let's focus on the styles of usage and when they're applicable like:

- SVG Sprites (when and why to use it)
- SVG embedded into HTML (when and why to use it)
- Image element or background image (when and why to use it)